packages/ui/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@vee-validate/zod": "^4.11.8",
     "bootstrap": "5.3.2",
+    "chart.js": "^4.4.0",
```

Comment:
not sure it makes sense to do this? As we dont do anything with charts why not just leave to prime vue

Reply:
It says on the docs that you need to install chart.js - https://primevue.org/chart/#chartjs